eeps/eep-0011.md

```diff
@@ -302,34 +302,34 @@ efficient than compiling each time one wants to match.
 
 The options have the following meanings:
 
-*   `anchored`  
+*   `anchored`
```

Comment:
You mean the whitespace characters?

Reply:
(because I see a newline in both the rendered HTML and Markdown)

Reply:
Trailing whitespaces are relevant in markdown. See https://spec.commonmark.org/0.30/#hard-line-break

For example:

```
<!-- Note the double space after Hello -->
1. Hello  
  world
```
is translated to
```html
<ol><li><p>Hello<br/>world</p></li></ol>
```
while

```
1. Hello

  world
```
is translated to
```html
<ol><li><p>Hello</p><p>world</p></li></ol>
```

The original EEP used the first style of double-space in order to create a newline without wrapping in `<p>` tags. It is not a big difference, but I think the first version looked better.

Reply:
TIL

Hm... I guess I can reverse this. Let me try to reverse e03a2d5. I won't actually argue on "what looks better", because I think that's a question of personal taste, of course. 😄